src/scoreList.js

```diff
@@ -1,14 +1,37 @@
-const listBoard = document.createElement('div');
-const array = [
-  { name: 'Firdavs', score: 91 },
-  { name: 'Said', score: 69 },
-  { name: 'Sediq', score: 51 },
-  { name: 'Amedzro', score: 48 },
-];
-
-for (const i of array) {
-  const p = document.createElement('p');
-  p.textContent = `${i.name}: ${i.score}`;
-  listBoard.appendChild(p);
+import { nameClass, scoreClass, array } from "./index.js";
+
+class List {
+  constructor(name, score) {
+    this.name = name;
+    this.score = score;
+  }
 }
-export { listBoard, array };
+
+const listBoard = document.createElement("div");
+const addListToPage = () => {
+  if (nameClass.value && scoreClass.value) {
+    const list = new List(nameClass.value, scoreClass.value);
+    array.push(list);
+    nameClass.value = null;
+    scoreClass.value = null;
+    func();
+  }
+};
+
+const func = async()=>{
+  const response = await fetch(
```

Comment:
It would be nice, if you give this function a descriptive name.
### :slightly_smiling_face: